MeshGeoToolsLib/GeoMapper.cpp

```diff
@@ -115,34 +138,54 @@ void GeoMapper::mapData()
 		min_val = bounding_box.getMinPoint()[2];
 		max_val = bounding_box.getMaxPoint()[2];
 	}
-	std::size_t nPoints (points->size());
 
-	if (!is_borehole)
+	std::vector<GeoLib::Point*> const* points (_geo_objects.getPointVec(_geo_name));
```

Comment:
you already get this vector at the caller site, so it could be passed as an argument. (not so important)